scripts/test_commands.sh

```diff
@@ -1,8 +1,14 @@
 #!/bin/bash
 set -e
 
-mkdir testNeuCLI
-cd testNeuCLI
+# cd to the directory of this script
+cd $(cd -P -- "$(dirname -- "$0")" && pwd -P)
+
+rm -rf ".tmp"
+mkdir ".tmp"
+cd ".tmp"
+
+tmpDirPath=$(pwd)
```

Comment:
Could you please resolve the conflicts -- then you will get `TMP_PATH` I guess :rocket: :tada: